### PR TITLE
storage: Reword Tang fingerprint verification dialog

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Review by jelly — COMMENTED** [review]
I assume we drop SHA1 is some mark it as insecure. Seems the tests still need to be adjusted to match the reworded test.

**Comment by mvollmer** [comment]
> I assume we drop SHA1 is some mark it as insecure. 

The thinking was that everybody can produce SHA256 fingerprints on the Tang server, and showing the SHA1 fingerprint doesn't have any value anymore, and just complicates the dialog.

**Comment by mvollmer** [comment]
> The thinking was that everybody can produce SHA256 fingerprints on the Tang server,

Yeah, except debian-stable, rhel-8, and centos-8, hmm...

**Comment by mvollmer** [comment]
Looks like this now:

image

**Review by jelly — APPROVED** [review]

**Comment by mjahoda** [comment]
I would change the wording a bit:

1. Check the key hash with the Tang server. -> Compare the key hash with the key hash of the Tang server.
2. In a terminal, run: -> In a terminal, enter:

File: pkg/storaged/crypto-keyslots.jsx [hunk 1]
```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
...
-            <>
-                <p>{_("Make sure the key hash from the Tang server matches one of the following:")}</p>
-
-                <h2 className="sigkey-heading">{_("SHA256")}</h2>
-                { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha256}</p>) }
-
-                <h2 className="sigkey-heading">{_("SHA1")}</h2>
-                { sigkey_thps.map(s => <p key={s} className="sigkey-hash">{s.sha1}</p>) }
+            <TextContent>
+                <Text component={TextVariants.p}>{_("Check the key hash with the Tang server.")}</Text>
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:627]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 2]
```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
...
 
-                <p>
-                    {_("Manually check with SSH: ")}
+                <Text component={TextVariants.h3}>{_("How to check")}</Text>
+                <Text component={TextVariants.p}>
+                    {_("In a terminal, run: ")}
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 3]
```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
...
                                    variant="inline-compact"
                                    isCode>
                         {cmd}
                     </ClipboardCopy>
-                </p>
-            </>
+                </Text>
+                <Text component={TextVariants.p}>
+                    {_("Check that the SHA-256 or SHA-1 hash from the command matches this dialog.")}
+                </Text>
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:641]: These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 4]
```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
...
+
+                <Text component={TextVariants.h3}>{_("SHA-256")}</Text>
+                { sigkey_thps.map(s => <Text key={s.sha256} component={TextVariants.pre}>{s.sha256}</Text>) }
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:644]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx [hunk 5]
```diff
@@ -623,25 +623,29 @@ function edit_tang_adv(client, block, key, url, adv, passphrase) {
...
+
+                <Text component={TextVariants.h3}>{_("SHA-1")}</Text>
+                { sigkey_thps.map(s => <Text key={s.sha1} component={TextVariants.pre}>{s.sha1}</Text>) }
+            </TextContent>
```
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:648]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:627]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:631]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:641]: These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 4 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:644]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/crypto-keyslots.jsx (hunk 5 above)
Comment (cockpituous) [inline pkg/storaged/crypto-keyslots.jsx:648]: These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18908-20230629-120408-c5437728-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**